<?xml version="1.0"?>
<launch>

	<arg name="robot" default="gofa" />

	<arg if="$(eval arg('robot') == 'gofa')" name="dice_pose" default="'{header: {seq: 0, stamp: {secs: 0, nsecs: 0}, frame_id: 'gofa_base_link'}, pose: {position: {x: 0.0663425968666783, y: -0.6218963996694468, z: 0.16700570241502155}, orientation: {x: 0.85544332495577, y: 0.5178958231656118, z: -0.0006812125851715791, w: 0.0004124144081274472}}}'"/>

	<arg if="$(eval arg('robot') == 'yumi')" name="dice_pose" default="'{header: {seq: 0, stamp: {secs: 0, nsecs: 0}, frame_id: 'yumi_base_link'}, pose: {position: {x: 0.0663425968666783, y: -0.3218963996694468, z: 0.16700570241502155}, orientation: {x: 0.85544332495577, y: 0.5178958231656118, z: -0.0006812125851715791, w: 0.0004124144081274472}}}'"/>

	<node pkg="rostopic" type="rostopic" name="fake_dice_spawner" args="pub -r 10 /dice_pose geometry_msgs/PoseStamped $(arg dice_pose)"/>

</launch>
